plugins/tech-radar/src/components/Radar/Radar.tsx

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 
-import React, { FC, useState, useRef } from 'react';
+import React, { useState, useRef } from 'react';
 import RadarPlot from '../RadarPlot';
-import { Ring, Quadrant, Entry } from '../../utils/types';
+import type { Ring, Quadrant, Entry } from '../../utils/types';
```

Comment:
@spotify/backstage-core  Don't we support absolute imports in plugins?

Reply:
Nop, we nuked it. Too much trouble getting it to work in a monorepo. It was pretty much either that or pointing to src/ for dev